### PR TITLE
docs: Slightly altering the debounced selector and effects example

### PR DESCRIPTION
**Pull request by h-jennings**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
The previous documentation example uses the `this.effect` API in a way that is inconsistent with the rest of the docs (actually not even sure if it is valid syntax).

## What is the new behavior?
The revised example in the docs now uses code patterns consistent with the `effects` doc section

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/content/guide/component-store/read.md [hunk 1]
```diff
@@ -115,7 +110,19 @@ export class MoviesStore extends ComponentStore&lt;MoviesState&gt; {
     (moviesPerPage, currentPageIndex) => ({moviesPerPage, currentPageIndex}),
     {debounce: true}, // 👈 setting this selector to debounce
   );
-}
+  
+  private readonly fetchMovies = this.effect(
+    (moviePageData$: Observable<{ moviesPerPage: number; currentPageIndex: number }>) => {
```
Comment (markostanimirovic) [inline projects/ngrx.io/content/guide/component-store/read.md:115]:
```suggestion
    (moviePageData$: Observable<{moviesPerPage: number; currentPageIndex: number}>) => {
```

File: projects/ngrx.io/content/guide/component-store/read.md [hunk 2]
```diff
@@ -115,7 +110,19 @@ export class MoviesStore extends ComponentStore&lt;MoviesState&gt; {
...
+      return moviePageData$.pipe(
+        concatMap(({ moviesPerPage, currentPageIndex }) => {
```
Comment (markostanimirovic) [inline projects/ngrx.io/content/guide/component-store/read.md:117]:
```suggestion
        concatMap(({moviesPerPage, currentPageIndex}) => {
```

File: projects/ngrx.io/content/guide/component-store/read.md [hunk 3]
```diff
@@ -115,7 +110,19 @@ export class MoviesStore extends ComponentStore&lt;MoviesState&gt; {
...
+          return this.movieService
+            .loadMovies(moviesPerPage, currentPageIndex)
+            .pipe(tap((results) => this.updateMovieResults(results)));
+        }),
+      );
+    },
+  );
+};
```
Comment (markostanimirovic) [inline projects/ngrx.io/content/guide/component-store/read.md:125]:
```suggestion
}
```

**Review by markostanimirovic — COMMENTED** [review]
Thanks @h-jennings! Just a few formatting suggestions to be consistent with the other code in this example.

**Review by markostanimirovic — APPROVED** [review]

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
Thanks @h-jennings!

**Comment by h-jennings** [comment]
@brandonroberts Happy to help! Thanks for everyone's patience and helpful feedback through the review process.

**Pull request merged**